haystack/nodes/label_generator/pseudo_label_generator.py

```diff
@@ -192,20 +194,20 @@ def generate_margin_scores(
 
     def generate_pseudo_labels(self, documents: List[Document], batch_size: Optional[int] = None) -> Tuple[dict, str]:
         """
-        Given a list of documents, generate a list of question-document pairs, mine for negatives, and
-        score positive/negative margin with cross-encoder. The output is the training data for the
+        Given a list of documents, this function generates a list of question-document pairs, mines for negatives, and
+        scores a positive/negative margin with cross-encoder. The output is the training data for the
         adaptation of dense retriever models.
 
-        :param documents: List[Document] = List of documents to mine negatives from
+        :param documents: List[Document] = The list of documents to mine negatives from.
         :type documents: List[Document]
-        :param batch_size: The number of documents to process in a batch
+        :param batch_size: The number of documents to process in a batch.
         :type batch_size: Optional[int]
         :return: A dictionary with a single key 'gpl_labels' representing a list of dictionaries, where each
         dictionary contains the following keys:
-            - question: the question
-            - pos_doc: the positive document for the given question
-            - neg_doc: the negative document for the given question
-            - score: the margin score (a float)
+            - question: The question string.
+            - pos_doc: Positive document for the given question.
+            - neg_doc: Negative document for the given question.
+            - score: The margin score the answer must fall within. A float.
```

Comment:
@agnieszka-m I wouldn't call the margin score "the answer must fall within" but just the CE(query, positive) - CE(query, negative) where CE is CrossEncoder.

Reply:
fixed